project-metrics/metrics_service/templates/metric_guage.html

```diff
@@ -0,0 +1,9 @@
+{% macro metric_guage(metric) -%}
+<div class='metric-guage metric-score-{{ metric.score.name.lower() }}'>
```

Comment:
Use double-quotes (") to delimit HTML attributes (here and in the other HTML file)

Reply:
Done